doc/mgr/administrator.rst

```diff
@@ -25,6 +25,15 @@ of ``ceph status``, which should now include a mgr status line::
 
     mgr active: $name
 
+Client authentication
+---------------------
+The manager is a new daemon which requires new CephX capabilities. If you upgrade
```

Comment:
The use of *"you"* and *"your"* can be removed, doesn't add anything specific to what authentication is trying to do. It would read like: 

    The manager is a daemon which requires CephX capabilities. When upgrading
    a cluster from an old version of Ceph, or when using the default install/deploy
    tools, the admin client should get this capability automatically. Using custom
    tooling may get ``EACCES`` errors when invoking certain ceph cluster commands.
    To fix that, add a "mgr allow *" stanza to the client's cephx capabilities by